test/config_parser_SUITE.erl

```diff
@@ -1761,17 +1761,16 @@ mod_http_upload(_Config) ->
              {token_bytes, 32}],
     ?eqf(modopts(mod_http_upload, MBase), T(Base)),
     ?errf(T(Base#{<<"host">> => -1})),
-    ?errf(T(Base#{<<"host">> => <<" f g ">>})),
     ?errf(T(Base#{<<"backend">> => <<"dev_null_as_a_service">>})),
-    ?errf(T(Base#{<<"expiration_time">> => <<>>})),
-    ?errf(T(Base#{<<"expiration_time">> => -1})),
+    ?errf(T(Base#{<<"expiration_time">> => 0})),
```

Comment:
I think we need to rewrite the tests to:
- Put only the required items in `Base`.
- Test remaining options one-by-one.
- Have a separate test case for `s3` if possible.

This can be done as a part of this PR or separately - the decision is up to you.